internal/cmdrepoget/command.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmdrepoget
+
+import (
+	"context"
+
+	"github.com/GoogleContainerTools/kpt/internal/errors"
+	"github.com/GoogleContainerTools/kpt/internal/util/porch"
+	configapi "github.com/GoogleContainerTools/kpt/porch/controllers/pkg/apis/porch/v1alpha1"
```

Comment:
The Kptfile schema is stored in https://github.com/GoogleContainerTools/kpt-functions-sdk/tree/master/go/api/kptfile/v1. Do you think we shall move the porch controllers/pkg/apis/porch to that repo now Or it can wait? I'm wondering if other code need to import the schema as well.

Reply:
Good question. My intuition is that we may want to have some consolidate place where all package manipulation APIs reside, I would not think to look for an API that defines server-side resources in a function SDK ... so for now I'd conclude 'not yet' and we can adjust as we go.

If I had to make recommendation now, I'd put Porch API in `pkg/api` in `kpt` repository.